### PR TITLE
adding sha video to readme

### PR DESCRIPTION
**Pull request by pdg744**
Pawel from Discord created a video walkthrough explaining this example. 
This PR adds a pointer to that video from the readme.

### CODE REVIEW

**Comment by tzerrell** [comment]
It's from 0.14 (or _maybe_ an intermediate commit; but if so one that looks quite similar to 0.14).

Let's go ahead and link this, but include a warning that it's from version 0.14. and our API has changed in some ways since then.

File: examples/sha/README.md [hunk 1]
```diff
@@ -6,3 +6,7 @@ Notable details:
 * We use `risc0_zkvm::sha` rather than general-purpose SHA code in the zkVM. This is because the RISC Zero zkVM includes SHA-specific accelerator code, and we therefore get _much_ better performance by using the native SHA code.
 * We could have passed the guest a `String` rather than a string literal for the same result.
 * Interpretting the output on the host side as a `risc0_zkp::core::sha::Digest` is useful for quality of life. We could also directly read the bytes written in the journal, but then we need to be careful about endianness, printing the bytes (`Digest` implements the `Display` trait), etc.
+
+For a video walkthrough of this example, we suggest [this video from Web3Dev - Web in Chains](https://www.youtube.com/watch?v=Nmxe8cnuO-0&list=PLcPzhUaCxlCjHQleuRhYN09zXgj6b_s-e&index=4).
+Thanks to Pawel from [Discord](https://discord.gg/risczero) for creating and sharing this content! 
+Note that the code snippets in this video are based on the 0.14 version of the zkVM.
```
Comment (tzerrell) [inline examples/sha/README.md:12]: Maybe include an explicit warning that things are different now:
```suggestion
Note that the code snippets in this video are based on the 0.14 version of the zkVM, and some APIs shown have changed in more recent versions.
```

**Review by tzerrell — APPROVED** [review]
Great!

**Review by KowalewskiPawel — APPROVED** [review]
Looks great! Thank you for mentioning my video :)

**Pull request merged**